source/agora/common/Hash.d

```diff
@@ -233,3 +203,65 @@ nothrow @nogc @safe unittest
     str.c2 = 'c';
     assert(hashFull(str) == abc_exp);
 }
+
+/*******************************************************************************
+
+    Hashes multiple arguments together
+
+    Params:
+        T = variadic argument types
+        args = the arguments
+
+    Returns:
+        the hash of all the arguments
+
+*******************************************************************************/
+
+public Hash hashMulti (T...)(T args) nothrow @nogc @safe
```

Comment:
```suggestion
public Hash hashMulti (T...)(auto ref T args) nothrow @nogc @safe
```

Reply:
If I make it `auto ref` here, then I can't seem to use `ref` in the foreach loop. I'm guessing I'm hitting a language limitation here?

Reply:
Ah, I think this is a limitation with `static foreach`.